shone/opacity/dace.py

```diff
@@ -169,7 +169,10 @@ def bin_files(members):
 
 
 def get_opacity_dir_path_molecule(isotopologue, linelist):
-    return glob(os.path.join('tmp', isotopologue + '__' + linelist + "*e2b"))[0]
+    find_e2b = glob(os.path.join('tmp', isotopologue + '__' + linelist + "*e2b"))
+    if len(find_e2b):
+        return find_e2b[0]
+    return glob(os.path.join('tmp', isotopologue + '__' + linelist + "*"))[0]
```

Comment:
This line 175 as well as the old line 179 do not catch the case that for some reason the linelist file/tmp folder do not exist, and instead give a non-descript IndexError.

Reply:
In the download-molecule example, I actually encounter an error on line 221: Unique_wavelengths doesn't seem to be defined.